tests/cmdline/commands/test_code.py

```diff
@@ -259,7 +259,7 @@ def test_code_duplicate_ignore(run_cli_command, aiida_code_installed, non_intera
 
 
 @pytest.mark.usefixtures('aiida_profile_clean')
-@pytest.mark.parametrize('sort', (True, False))
+@pytest.mark.parametrize('sort', ['--sort', '--no-sort'])
```

Comment:
This change is not correct I believe. Now the test always runs with `--sort` since on line 285 you test for truthness of a string which is always `True`.

Reply:
Good catch! I wanted to be consistent with the `['--sort', '--no-sort']` parametrization for `test_computer.py`. Though, now that I looked at it again, I'm not sure that it does what it's supposed to do even there... For the sake of this PR, it's probably best to just revert this change.

Reply:
> Though, now that I looked at it again, I'm not sure that it does what it's supposed to do even there...

It does use it correctly in `test_computer_export_setup`, although it also parametrizes `test_computer_export_config` but then doesn't appear to use it, so that would be a bug.

As for changing the code export test. You would simply add it directly to the options, e.g. something like:
```python
@pytest.mark.usefixtures('aiida_profile_clean')
@pytest.mark.parametrize('sort_option', ['--sort', '--no-sort'])
def test_code_export(run_cli_command, aiida_code_installed, tmp_path, file_regression, sort_option):
    ...
    options = [str(code.pk), str(filepath), sort_option]
    # options.append('--sort' if sort else '--no-sort')

    run_cli_command(cmd_code.export, options)
```